### PR TITLE
Modals Ratings component

### PR DESCRIPTION
**Pull request by dev-enforced**
This Pull Request has the previews of the modals and ratings component along with the code added here. Utility classes for flex are being developed and will be added here to optimise the CSS code

### CODE REVIEW

File: components/modals.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.btn-modal-close {
+  position: absolute;
+  right: 1rem;
+  top: 0.5rem;
+  background-color: transparent;
+}
+
+@media screen and (max-width: 480px) {
```
Comment (Manju-sree) [inline components/modals.css:60]: good to see you have used media queries

File: components/ratings.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  font-size: 1.5rem;
+  background-color: var(--warning-bg);
+}
+
+.gentle-feedback-container {
+  background-color: transparent;
+  color: var(--text-color);
+}
+
+.positive:hover {
```
Comment (Manju-sree) [inline components/ratings.css:17]: please check this colors are not reflecting on UI, as it in black bg.

**Review by Manju-sree — APPROVED** [review]
good to see interactive rating components. code looks fine to merge

File: components/ratings.css (hunk 2 above)
Comment (dev-enforced) [reply components/ratings.css:17]: image
It is working as expected I used the chrome dev tools for checking the styles on hover

Comment (Manju-sree) [reply components/ratings.css:17]: fine it's working on hover.

**Review by mariyasada — APPROVED** [review]
looks good

**Pull request merged**